changelogs/fragments/820_nagios_added_acknowledge_and_servicecheck.yml

```diff
@@ -0,0 +1,3 @@
+minor_changes:
+- nagios - add the ``acknowledge`` parameter (https://github.com/ansible-collections/community.general/pull/820).
+- nagios - add the ``service_check`` parameter (https://github.com/ansible-collections/community.general/pull/820).
```

Comment:
These are not parameters, but actions, I think? From your change, it does not look like you are adding new module options (parameters).

Reply:
you're right, my mistake.
I'll correct it right away